### PR TITLE
Karkakol/update dependencies

### PR DESCRIPTION
**Pull request by karkakol**

### CODE REVIEW

File: .github/actions/setup/action.yml [hunk 1]
```diff
@@ -5,9 +5,10 @@ runs:
   using: composite
   steps:
     - name: Setup Node.js
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
         node-version-file: .nvmrc
```
Comment (karkakol) [inline .github/actions/setup/action.yml:10]: it can produce bugs on github actions

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -151,20 +152,24 @@ android {
...
-    debugImplementation("com.facebook.flipper:flipper-fresco-plugin:${FLIPPER_VERSION}")
+    implementation("androidx.swiperefreshlayout:swiperefreshlayout:1.0.0")
+  implementation 'androidx.core:core-ktx:+'
+
+//     debugImplementation("com.facebook.flipper:flipper:${FLIPPER_VERSION}")
+//     debugImplementation("com.facebook.flipper:flipper-network-plugin:${FLIPPER_VERSION}") {
+//         exclude group:'com.squareup.okhttp3', module:'okhttp'
+//     }
+//
+//     debugImplementation("com.facebook.flipper:flipper-fresco-plugin:${FLIPPER_VERSION}")
```
Comment (karkakol) [inline example/android/app/build.gradle:166]: remove unnecesary comments

File: example/ios/Podfile [hunk 3]
```diff
@@ -68,16 +68,16 @@ target 'JellyfishExample' do
...
     # Hermes is now enabled by default. Disable by setting this flag to false.
     # Upcoming versions of React Native may rely on get_default_flags(), but
     # we make it explicit here to aid in the React Native upgrade process.
-    :hermes_enabled => flags[:hermes_enabled],
-    :fabric_enabled => flags[:fabric_enabled],
+##     :hermes_enabled => flags[:hermes_enabled],
+##     :fabric_enabled => flags[:fabric_enabled],
     # Enables Flipper.
     #
     # Note that if you have use_frameworks! enabled, Flipper will not work and
```
Comment (karkakol) [inline example/ios/Podfile:83]: remove unnecesary comments, moreover consider removing flipper

**Review by karkakol — COMMENTED** [review]
Add:
1. info about prerequirements in android

- service for android 14+ in android manifest
- link to react native 0.73 and expo 50
2. Info about autogenerating folder service api
3. Inspect why after update lints and build does not work
4. Remember about react native membrane dependency
- release it before merging
- update all dependencies there
5. Run more tests on physical devices with different os system

**Review by graszka22 — APPROVED** [review]

**Pull request merged**